common/common.go

```diff
@@ -170,7 +170,7 @@ func (l *PrivateNetList) Strings() []string {
 }
 
 func (l *PrivateNetList) Any() bool {
-	return len(l.mapping) > 0
+	return len(l.mapping) > 0 && !l.mapping["false"]
```

Comment:
My preference is to go the other way. Treat `false` and `true` as just network names and join all networks is just `--private-net`. Or at least `--private-net=all`. If the arg is a list, special value of `all` makes sense but `true` is an oddity.